src/easylink/pipeline_graph.py

```diff
@@ -516,10 +480,29 @@ def _update_slot_filepaths(self, config: Config) -> None:
                         str(
                             Path("intermediate")
                             / node
+                            # embarrassingly parallel implementations rely on snakemake wildcards
+                            # TODO: [MIC-5787] - need to support multiple wildcards at once
+                            / ("{chunk}" if implementation.is_embarrassingly_parallel else "")
```

Comment:
This is where the (currently only one and hardcoded as "{chunk}") wildcards enter the mix for embarrassingly parallel snakemake rules (i.e. checkpoints)